main/main.c

```diff
@@ -339,10 +336,6 @@ main(int argc, char **argv)
 	};
 	int ret, log_fd;
 
-	gatekeeper_logtype = rte_log_register("gatekeeper");
-	if (gatekeeper_logtype < 0)
-		rte_exit(EXIT_FAILURE, "Error registering gatekeeper log type\n");
-
```

Comment:
This seems the proper place to set `BLOCK_LOGTYPE` to `DEBUG`. Add the following comment to the new code:
```
/* Functional blocks have their own log level, and all log entries of Gatekeeper use BLOCK_LOGTYPE. Thus, setting BLOCK_LOGTYPE to DEBUG level instructs DPDK to log everything that the blocks decide to log. */
```
Break the comment in multiple lines as needed.